cmd/koord-scheduler/app/server.go

```diff
@@ -191,11 +191,13 @@ func Run(ctx context.Context, cc *schedulerserverconfig.CompletedConfig, sched *
 	cc.InformerFactory.Start(ctx.Done())
 	cc.KoordinatorSharedInformerFactory.Start(ctx.Done())
 	cc.NRTSharedInformerFactory.Start(ctx.Done())
+	cc.PodGroupInformerFactory.Start(ctx.Done())
 
 	// Wait for all caches to sync before scheduling.
 	cc.InformerFactory.WaitForCacheSync(ctx.Done())
 	cc.KoordinatorSharedInformerFactory.WaitForCacheSync(ctx.Done())
 	cc.NRTSharedInformerFactory.WaitForCacheSync(ctx.Done())
+	cc.PodGroupInformerFactory.WaitForCacheSync(ctx.Done())
```

Comment:
if cluster not import PodGroup, this step will hang, so should we add a flag to active by need?

Reply:
Ok,i will check how NRT CRD in Koordinator solve this problem

Reply:
When installing koordinator via helm charts it will import the PodGroup CRD's